org.eclipse.xtext.idea.common.types.tests/testdata/org/eclipse/xtext/common/types/testSetups/TypeWithInnerAnnotation.java

```diff
@@ -12,4 +12,5 @@
  */
 public class TypeWithInnerAnnotation {
 	public @interface MyAnnotation {}
+	void m(TypeWithInnerAnnotation this, int m) {}
```

Comment:
We should have a test with an annotation on the 'this' arg, but that's out of scope for this PR.